cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -2039,6 +2050,13 @@ fn x64_get_regs(inst: &Inst, collector: &mut RegUsageCollector) {
         | Inst::Fence { .. } => {
             // No registers are used.
         }
+
+        Inst::ElfTlsGetAddr { .. } => {
+            // All caller-saves are clobbered.
+            for reg in X64ABIMachineSpec::get_regs_clobbered_by_call(CallConv::SystemV) {
```

Comment:
Double-check me here: we can hardcore SystemV here because this instruction is specific for Elf, right? If that's not the case, could we get the call_conv from the caller; if that's the case, can you add a comment to that effect please?

Reply:
Yes, this instruction is specific for ELF and must use the System-V call conv independently of the call conv of the function using TLS. Mach-O and PE use different TLS mechanisms.

Reply:
Comment added!